samples/openthread/coprocessor/README.rst

```diff
@@ -139,76 +140,104 @@ Testing
-        "Config:NCP:DriverName" => "spinel"
-        "NCP:HardwareAddress" => [E8F947748F493141]
-      ]
+         The output will look similar to the following:
 
-   This output means that NCP is offline.
-#. In the wpanctl shell, run the following command to set up a Thread network:
+         .. code-block:: console
 
-   .. code-block:: console
+            leader_if => [
+              "NCP:State" => "offline"
+              "Daemon:Enabled" => true
+              "NCP:Version" => "OPENTHREAD/gde3f05d8; NONE; Jul  7 2020 10:04:51"
+              "Daemon:Version" => "0.08.00d (0.07.01-343-g3f10844; Jul  2 2020 09:07:40)"
+              "Config:NCP:DriverName" => "spinel"
+              "NCP:HardwareAddress" => [E8F947748F493141]
+            ]
 
-      wpanctl:leader_if> form "My_OpenThread_network"
+         This output means that NCP is offline.
+      #. In the wpanctl shell, run the following command to set up a Thread network:
 
-   The output will look similar to the following:
+         .. code-block:: console
 
-   .. code-block:: console
+            wpanctl:leader_if> form "My_OpenThread_network"
 
-      Forming WPAN "New_network" as node type "router"
-      Successfully formed!
+         The output will look similar to the following:
 
-   This output means that the network was formed successfully.
-#. In the wpanctl shell, run the status command again to see that "My_OpenThread_network" was formed by NCP:
+         .. code-block:: console
 
-   .. code-block:: console
+            Forming WPAN "New_network" as node type "router"
+            Successfully formed!
+
+         This output means that the network was formed successfully.
+      #. In the wpanctl shell, run the status command again to see that "My_OpenThread_network" was formed by NCP:
+
+         .. code-block:: console
+
+            wpanctl:leader_if> status
+
+      The final output will be similar to the following:
+
+      .. code-block:: console
+
+         leader_if => [
+           "NCP:State" => "associated"
+           "Daemon:Enabled" => true
+           "NCP:Version" => "OPENTHREAD/gde3f05d8; NONE; Jul  7 2020 10:04:51"
+           "Daemon:Version" => "0.08.00d (0.07.01-343-g3f10844; Jul  2 2020 09:07:40)"
+           "Config:NCP:DriverName" => "spinel"
+           "NCP:HardwareAddress" => [E8F947748F493141]
+           "NCP:Channel" => 26
+           "Network:NodeType" => "leader"
+           "Network:Name" => "My_OpenThread_network"
+           "Network:XPANID" => 0x048CA9024CD7D40F
+           "Network:PANID" => 0xDB92
+           "IPv6:MeshLocalAddress" => "fd04:8ca9:24c:0:ebb8:4ef3:d96:c4bd"
+           "IPv6:MeshLocalPrefix" => "fd04:8ca9:24c::/64"
+           "com.nestlabs.internal:Network:AllowingJoin" => false
+         ]
+
+      This output means that you have successfully formed the Thread network.
+
+      .. _ot_coprocessor_sample_testing_more_boards:
+
+   .. tab:: b. RCP
```

Comment:
I recommend putting here the `.. tab:: b. RCP with wpantund` and mention that the output will be exactly the same as above. The only difference is the different wpantund configuration which needs to point to the `ot-ncp` app instead of the pure serial interface.

I would rename below to something like `.. tab:: b. RCP with ot-deamon` and also mention the `ot-cli`.

To sum up, there are three options:
1. RCP with the `ot-ncp` and `wpantund` which works exactly like the NCP, not covered in this readme.
2. RCP with the `ot-daemon` and `ot-ctl` which works exactly like the CLI, the daemon is in the background all the time, covered in this readme.
3. RCP with the `ot-cli` which works exactly like the CLI, except the fact nothing runs in the background, not covered in this readme at all.

Maybe we should at lest mention all of these configuration for the sake of completeness. What do you think?

Reply:
Because both NCP and RCP can use wpantund I decided to simplify description of this sample and base on wpantund reference. I've added references to Thread tools ot-daemon and ot-cli instead in Thread User Guide.